ci-operator/config/openshift/azure-file-csi-driver-operator/openshift-azure-file-csi-driver-operator-release-4.10.yaml

```diff
@@ -54,12 +54,44 @@ tests:
   optional: true
   steps:
     cluster_profile: azure4
-    workflow: openshift-e2e-azure-csi-file
+    env:
+      CLUSTERCSIDRIVER: file.csi.azure.com
+      TEST_CSI_DRIVER_MANIFEST: manifest-azure-file.yaml
+      TEST_SUITE: openshift/csi
+      TRUECONDITIONS: AzureFileDriverControllerServiceControllerAvailable AzureFileDriverNodeServiceControllerAvailable
+    post:
+    - ref: storage-obj-check
+    - chain: gather-core-dump
+    - chain: ipi-azure-post
+    pre:
+    - chain: ipi-azure-pre
+    - ref: ipi-conf-techpreview
```

Comment:
I think ipi-conf-techpreview needs to be before ipi-azure-pre, as it only creates manifest for the installer.